Projects/hGraphQL/Sources/Models/Claims.swift

```diff
@@ -200,3 +200,20 @@ public struct ClaimData {
         claims = cardData.claimsStatusCards.map { .init(cardData: $0) }
     }
 }
+
+extension Claim: Equatable {
+    public static func ==(lhs: Claim, rhs: Claim) -> Bool {
+        return lhs.id == rhs.id
```

Comment:
Why not make the whole object equatable, why just check the id? Doing this usually leads to weird behaviours I've seen 😬